PowerEditor/src/NppDarkMode.cpp

```diff
@@ -1547,6 +1740,43 @@ namespace NppDarkMode
 					SetWindowTheme(hwnd, p.themeClassName, nullptr);
 				}
 
+				auto style = ::GetWindowLongPtr(hwnd, GWL_STYLE);
+				bool isComboBox = (style & LBS_COMBOBOX) == LBS_COMBOBOX;
+				auto exStyle = ::GetWindowLongPtr(hwnd, GWL_EXSTYLE);
+				bool hasClientEdge = (exStyle & WS_EX_CLIENTEDGE) == WS_EX_CLIENTEDGE;
+
+				if (p.subclass && !isComboBox && hasClientEdge)
+				{
+					NppDarkMode::subclassCustomBorderForListBoxAndEditControls(hwnd);
+				}
+
+#ifndef __MINGW64__ // mingw build for 64 bit has issue with GetWindowSubclass, it is undefined
```

Comment:
@ozone10 
Could you try `#define _WIN32_WINNT 0x502` instead?
Ref: https://stackoverflow.com/questions/60105064/setwindowsubclass-function-from-winapi

Reply:
Unfortunately it cause more issue. It needs higher number, but that also did not work.
I also tried `#define _WIN32_IE 0x0600`. 

But this is minor issue that code is mainly to avoid border flickering  when quickly moving to and out listbox.

Reply:
It's there in header but for some reason it's not used.

![image](https://user-images.githubusercontent.com/55940305/171684724-31f640ca-83a7-4d18-9ed6-8aa2f4198203.png)